src/Symfony/Component/PropertyAccess/Mapping/AttributeMetadata.php

```diff
@@ -0,0 +1,177 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\PropertyAccess\Mapping;
+
+/**
+ * {@inheritdoc}
+ *
+ * @author Kévin Dunglas <dunglas@gmail.com>
+ */
+class AttributeMetadata implements AttributeMetadataInterface
```

Comment:
This should be named PropertyMetadata to keep a consistent language with the rest of the component. Naming the same thing "property" once and "attribute" another time is confusing.